script.js

```diff
@@ -1,4 +1,123 @@
+var gameModeDiceRoll = "game_state_dice_roll";
+var gameStateChooseDiceOrder = "game_state_choose_dice";
+var gameCompareScore = "game_compare_score";
+var currentGameMode = gameModeDiceRoll;
+
+var currentPlayer = 1;
+// var userName = "";
+var allPlayerSCore = [];
+
+//create an array to store the values of the 2 dices for both players
+var currentPlayerRolls = [];
+
+//creating helper function for dice roll
+var rollDice = function () {
+  var randomDecimal = Math.random() * 6;
+  var randomInteger = Math.floor(randomDecimal);
+  var diceNumber = randomInteger + 1;
+  return diceNumber;
+};
```

Comment:
this was shown step by step in the preclass to simplify and make it easier to understand but consider condensing all of them into one line in your code

return Math.floor(Math.random() * 6) + 1